src/test/java/guitests/guihandles/PersonListPanelHandle.java

```diff
@@ -114,7 +114,7 @@ public PersonCardHandle navigateToPerson(String name) {
     /**
      * Navigates the listview to display and select the person.
      */
-    public PersonCardHandle navigateToPerson(ReadOnlyTask person) {
+    public TaskCardHandle navigateToPerson(ReadOnlyTask person) {
```

Comment:
Do we stick to navigateToPerson method name for now or to change it to navigateToTask?